modules/ROOT/examples/live-demos/inline-css/index.js

```diff
@@ -0,0 +1,45 @@
+const settings = {
+  plugins: [
+    "advlist", "anchor", "autolink", "charmap", "code", "fullscreen",
+    "help", "image", "insertdatetime", "link", "lists", "media",
+    "preview", "searchreplace", "table", "visualblocks", "inlinecss"
+  ],
+  toolbar: "undo redo | styles | bold italic underline strikethrough | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link image",
+  content_style: `
+    .red {
+      color: red;
+    }
+    .blue {
+      color: blue;
+    }
+    .blue-background {
+      background-color: blue;
+    }
+    .red-background {
+      background-color: red;
+    }
+  `
+};
+
+tinymce.init({
+  selector: "textarea.classic",
+  ...settings
+});
+
+const outputIframe = document.getElementById('outputIframe');
+const rawDoc = outputIframe.contentWindow.document;
```

Comment:
At least for me, the output Iframe is not filled with the content on Firefox if `rawdoc` and `outputIframe` are defined here. If you move them into the button-fired event (e.g. line 38) it does work though. [Fiddle](https://fiddle.tiny.cloud/driaab/15#) (line 210) for reference

Reply:
@Skylite73 We will have DX look into this.

Reply:
Suggested change committed, but still did not fix the issue within Firefox.

Reply:
This has been captured so it can be raised elsewhere and elsewhen.